decidim-core/app/views/decidim/profiles/show.html.erb

```diff
@@ -1,3 +1,11 @@
 <% add_decidim_page_title(t(@title_key, scope: "decidim.profiles.show")) %>
 
 <%= cell "decidim/profile", profile_holder, context: { content_cell: @content_cell } %>
+
+<%=
+  render partial: "decidim/shared/flag_user_modal", locals: {
+      reportable: profile_holder,
+      form: report_form,
+      url: decidim.report_user_path(sgid: profile_holder.to_sgid.to_s)
+  }
```

Comment:
Shouldn't this be rendered by `decidim-core/app/cells/decidim/author/flag_user.erb`?

Reply:
The tricky part is that report_form, that is being initialized in the controller part.